extensions/rocksdb-repos/FlowFileRepository.cpp

```diff
@@ -200,6 +200,20 @@ bool FlowFileRepository::initialize(const std::shared_ptr<Configure> &configure)
   }
   logger_->log_debug("NiFi FlowFile Repository Directory %s", directory_);
 
+  compaction_period_ = DEFAULT_COMPACTION_PERIOD;
+  if (auto compaction_period_str = configure->get(Configure::nifi_flowfile_repository_compaction_period)) {
+    if (auto compaction_period = TimePeriodValue::fromString(compaction_period_str.value())) {
+      compaction_period_ = compaction_period->getMilliseconds();
+      if (compaction_period_.count() == 0) {
+        logger_->log_warn("Setting '%s' to 0 disables forced compaction", Configure::nifi_dbcontent_repository_compaction_period);
+      }
+    } else {
+      logger_->log_error("Malformed property '%s', expected time period, using default", Configure::nifi_flowfile_repository_compaction_period);
+    }
+  } else {
+    logger_->log_info("Using default compaction period");
```

Comment:
I think this can use a lower log level. It would also be nice to include the value of this default.
```suggestion
    logger_->log_debug("Using default compaction period of %" PRId64 " ms", compaction_period_.count());
```

Reply:
changed it in both repositories